sdk/nodejs/storage/zMixins.ts

```diff
@@ -371,6 +376,13 @@ export interface QueueEventSubscriptionArgs extends appservice.CallbackFunctionA
      * be used in their place.
      */
     hostSettings?: QueueHostSettings;
+
+    /**
+     * Specify 'string' to get messages as a string, or a parsed object if the message is a valid JSON literal.
+     * Specify 'binary' for all data to be passed in as a Buffer.
+     * Default is 'string'.
+     */
+    dataType?: "binary" | "string";

```

Comment:
shouldn't this be `pulumi.Input<...>`?

Reply:
We could... Not all arguments are `Input`s and I can't think of a good example when this would be useful here. But we could.